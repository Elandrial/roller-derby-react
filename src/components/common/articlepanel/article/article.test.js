import Article from './article';
import renderer from 'react-test-renderer';

describe('Pager component',() => {
  it('renders correctly', () =>{
    const tree = renderer
    .create(<Article id={-1} title={"test case"} date={"1st April 2023"} description={"test example"} image={"derby-program.jpg"}/>)
    .toJSON();
  expect(tree).toMatchSnapshot();
  })
});