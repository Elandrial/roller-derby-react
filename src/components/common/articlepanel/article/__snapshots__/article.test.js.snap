// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pager component renders correctly 1`] = `
<article
  className="detail-card"
>
  <figure>
    <img
      alt="test case"
      src="/images/derby-program.jpg"
    />
  </figure>
  <div>
    <h3>
      test case
    </h3>
    <time>
      1st April 2023
    </time>
    <p>
      test example
    </p>
    <button
      onClick={[Function]}
    >
      Read More
    </button>
  </div>
</article>
`;
